src/libecalc/presentation/yaml/mappers/component_mapper.py

```diff
@@ -270,7 +271,19 @@ def from_yaml_to_dto(self, data: Dict) -> dto.Installation:
             try:
                 venting_emitters.append(YamlVentingEmitter(**venting_emitter))
             except ValidationError as e:
-                raise DtoValidationError(data=data, validation_error=e) from e
+                error_names = [err["loc"][0] for err in e.errors()]
```

Comment:
This does not seem venting emitter specific. If we are changing how we display messages we can do it inside DtoValidationError.

DtoValidationError is also handled, so no need to use EcalcError here :)

Reply:
Ah, of course. Nice:-)